.gitignore

```diff
@@ -115,4 +118,4 @@ env/
 ### Configuration files ###
 .env
 
-secrets.py
\ No newline at end of file
+secrets.py
```

Comment:
Why is this here?

Reply:
oh sorry in old way I wasn't using env vars and was excluding secrets.py for other exercise, I guess it doesn't matter this time